EmailNotificationEngine.cs

```diff
@@ -697,7 +708,17 @@ private void ProcessWatchers(List<IssueDto> issues, DateTime lastChecked)
                     }
                 }
             }
-        }       
+        }
+
+        private int? GetLanguageId( UserDto recipient )
+        {
```

Comment:
Gets the user's language, eg FR-fr, but if that didn't exist, look for the first French one, eg FR-ca